pkgs/applications/editors/bluej-bin/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchurl, makeWrapper, unzip, jetbrains
```

Comment:
I didn't notice jetbrains as an explicit dependency on the website. Can you please elaborate on your choice?

Reply:
Well, this should've been `jetbrains-jdk` not `jetbrains`. BlueJ needs JavaFX, so no openjdk here (#50418).
Will fix this.

Reply:
Ah ok that makes sense.